Test/DatabaseTests/TestDatabaseLoaders.cs

```diff
@@ -197,7 +199,7 @@ public void FilesEqualHash()
 
             // UniProt PTMs
             var uniprotPtms = Loaders.LoadUniprot(uniProtPath, formalChargesDictionary).ToList();
-            Assert.AreEqual(347, uniprotPtms.Count()); // UniProt PTM list may be updated at some point, causing the unit test to fail
+            Assert.AreEqual(359, uniprotPtms.Count()); // UniProt PTM list may be updated at some point, causing the unit test to fail
```

Comment:
Do we need to test the exact number? Could we test that it's greater than 300 so we don't have to update it?